utils/markdown/index.test.js

```diff
@@ -107,6 +105,14 @@ this is long text this is long text this is long text this is long text this is
 ~~~`);
 
         expect(output).toContain('<span class="post-code__language">TeX</span>');
-        expect(output).toContain('<code class="hljs hljs-ln">');
+    });
+
+    test('should add line numbers to code', () => {
```

Comment:
should we add a test to make sure it fixes the reported case?
````
```python
    op.execute("""
        UPDATE events.settings
        SET name = 'paper_review_conditions'
        WHERE module = 'editing' AND name = 'review_conditions'
    """)
```
````